test/cql-pytest/api_util.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+
+# This file provides utility to REST API requests.
```

Comment:
Nitpick: I would call this file rest_api or something like that, because the word "api" is too overloaded (CQL is also an API...) and the word "util" doesn't add anything.